tests/probes/rpmverifyfile/test_probes_rpmverifyfile_older.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<oval_definitions xsi:schemaLocation="http://oval.mitre.org/XMLSchema/oval-definitions-5 oval-definitions-schema.xsd      http://oval.mitre.org/XMLSchema/oval-definitions-5#unix unix-definitions-schema.xsd" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://oval.mitre.org/XMLSchema/oval-definitions-5" xmlns:oval-def="http://oval.mitre.org/XMLSchema/oval-definitions-5" xmlns:oval="http://oval.mitre.org/XMLSchema/oval-common-5" xmlns:unix-def="http://oval.mitre.org/XMLSchema/oval-definitions-5#unix" xmlns:lin-def="http://oval.mitre.org/XMLSchema/oval-definitions-5#linux">
+  <generator>
+    <oval:schema_version>5.11</oval:schema_version>
+    <oval:timestamp>2015-01-12T10:41:00-05:00</oval:timestamp>
+  </generator>
+  <definitions>
+    <definition id="oval:x:def:1" version="1" class="miscellaneous">
+      <metadata>
+        <title>Test rpmverifyfile probe</title>
+        <description>Basic test of implementation of the rpmverifyfile_object.</description>
+      </metadata>
+      <criteria>
+        <criterion comment="Test that all files are from /etc directory." test_ref="oval:x:tst:1"/>
+      </criteria>
+    </definition>
+  </definitions>
+
+  <tests>
+    <lin-def:rpmverifyfile_test id="oval:x:tst:1" version="1" comment="Test" check="all">
+      <lin-def:object object_ref="oval:x:obj:1"/>
+    </lin-def:rpmverifyfile_test>
+  </tests>
+
+  <objects>
+    <lin-def:rpmverifyfile_object id="oval:x:obj:1" version="1" comment="Object">
+        <lin-def:behaviors nolinkto="true" nosize="true" nouser="true" nogroup="true" nomtime="true" nomode="true" nordev="true" noconfigfiles="true" noghostfiles="true" nomd5="true"/>
+        <lin-def:name operation="pattern match"/>
+        <lin-def:epoch operation="pattern match"/>
+        <lin-def:version operation="pattern match"/>
+        <lin-def:release operation="pattern match"/>
+        <lin-def:arch operation="pattern match"/>
+        <lin-def:filepath>/etc/redhat-release</lin-def:filepath>
```

Comment:
I don't like that we are relying on `/etc/redhat-release` in an upstream test. Could we choose a different file? Could we skip the test if /etc/redhat-release is not available?